recipes/islpy/meta.yaml

```diff
@@ -0,0 +1,77 @@
+{% set name = "islpy" %}
+{% set version = "2016.2.1" %}
+{% set sha256 = "b23ccca253945ba7b41d091e86b3e0ace6a256238f28abf604de1d06f878895c" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt && cd test && python -m pytest
```

Comment:
The test part should be in a `run_test.py` script to disambiguate `build` from `test`.